inc/profile.class.php

```diff
@@ -45,6 +45,10 @@ class PluginFlyvemdmProfile extends Profile {
     */
    static $rightname = 'flyvemdm:flyvemdm';
 
+   /**
+    * Deletes the profiles
```

Comment:
use your other nice description of callback for hooks

Reply:
Ok, I'll fix them now